ui/src/support/constants.ts

```diff
@@ -0,0 +1,25 @@
+import { generateRandomString } from "./utils"
+
+export const BASE_URL = "https://about.gitlab.com/"
+export const SIGN_UP_URL = "https://gitlab.com/-/trial_registrations/new"
+export const defaultTimer = 50
+export const searchButton = `//button[contains(@aria-label ,'Search')]`
+export const confirmationMessage = `//div[contains(text(), "You must confirm your email within 1 days of signing up. If you do not confirm your email in this timeframe, your account will be deleted and you will need to sign up for GitLab again.")]`
```

Comment:
It is a locator. Please name the constant accordingly.

Reply:
done